tests/headers/structs.h

```diff
@@ -29,7 +29,7 @@ struct recursive_struct {
 typedef struct tagWNDCLASSEXA {
     int         cbClsExtra;
     int         cbWndExtra;
-} WNDCLASSEXA, *PWNDCLASSEXA, near *NPWNDCLASSEXA, far *LPWNDCLASSEXA;
```

Comment:
Neither __far, __near nor far, near are supported by up-to-date visual compiler versions.
This is a legacy from the days when compilers generated 16bit code.

If you have a look into the windows header files, you'll see, that they define these identifiers to nothing:
```C
#define far
```